### PR TITLE
Link to b.p.o when both issue number and trivial label are found

### PR DESCRIPTION
**Pull request by Mariatta**
Fixes https://github.com/python/bedevere/issues/4

### CODE REVIEW

File: bedevere/bpo.py [hunk 1]
```diff
@@ -55,11 +51,27 @@
...
 
 @router.route("pull_request", "unlabeled")
 async def removed_label(gh, event):
     """Re-check the status if the "trivial" label is removed."""
     if event.data["label"]["name"] == TRIVIAL_LABEL:
         await set_status(gh, event)
+
+
+def get_found_issue_number_status(found_issue):
+    """Get the status for when an issue number was found in the title"""
```
Comment (brettcannon) [inline bedevere/bpo.py]: Missing a period and you're creating a status instead of getting one.

File: bedevere/bpo.py [hunk 2]
```diff
@@ -55,11 +51,27 @@
...
 
 
 @router.route("pull_request", "unlabeled")
 async def removed_label(gh, event):
     """Re-check the status if the "trivial" label is removed."""
     if event.data["label"]["name"] == TRIVIAL_LABEL:
         await set_status(gh, event)
+
+
+def get_found_issue_number_status(found_issue):
```
Comment (brettcannon) [inline bedevere/bpo.py]: You're actually not "getting" a status but creating one. So maybe `create_success_status()` (technically ambiguous due to trivial issues, but they have their own static status so I think it's fine)?

Comment (Mariatta) [inline bedevere/bpo.py]: Thanks I renamed this :)

**Comment by Mariatta** [comment]
🤔 Anything I need to do about codecov?

**Comment by brettcannon** [comment]
@Mariatta don't worry about codecov; it was turned on after this PR was created so it's just in a weird state.

**Pull request merged**